docs/knnsridhar.md

```diff
@@ -0,0 +1,5 @@
+# knnsridhar Bio
+
+- **Name:** GitHub Student
+- **Occupation:** Student
+- **Location:** SFO
```

Comment:
Is this San Francisco?  I think that's what you mean, but I'm not completely sure. :smiley: